tests/test_cmds.py

```diff
@@ -53,10 +53,10 @@ def test_upload(runner):
     res = runner.invoke(root, cmd.split())
     assert res.exit_code == 0
 
-#def test_search(runner):
-#    cmd = "-b ngs-test -c {0} search -q {1}".format(credentials_path, f1target)
-#    res = runner.invoke(root, cmd.split())
-#    assert res.exit_code == 0 
+def test_search(runner):
+    cmd = "-b ngs-test -c {0} search -q {1}".format(credentials_path, f1target)
```

Comment:
Might be a future issue if ngs-test gets moved or emptied etc

Reply:
Clarified in source